ush/config_defaults.yaml

```diff
@@ -886,6 +895,9 @@ workflow:
 
   CYCL_HRS_SPINSTART: []
   CYCL_HRS_PRODSTART: []
+  BOUNDARY_LEN_HRS: 0
+  BOUNDARY_LONG_LEN_HRS: 0
```

Comment:
This one isn't used in the logic here at all as far as I can tell. Does it need to be?

Reply:
It is used in the workflow for LBCS task just like LBCS_SEARCH_HRS.